.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,16 +19,14 @@ For a PR to be considered, each item in the checklist must be checked.
 <!-- To record and share a video: http://recordit.co/ -->
 
 ### How to test
-<!--
-Please describe the steps for reviewers to take to cover all facets of this feature
+<!-- Please describe the steps for reviewers to take to cover all facets of this feature. -->
 
-1. `npm run clean && npm install`
-2. navigate to Site > Button > Buttons with kittens section
-3. Click the Cat Button twice (three times in Internet Explorer)
+1. `cd mineral-ui/packages/button && npm start`
+2. Navigate to Site > Button > Buttons with kittens section
+3. Click the Cat Button twice (three times in IE 11)
 4. ¡Hear the cat meow en Español!
 
--->
-<!-- Please describe in detail how you tested your changes beyond automated tests. -->
+<!-- Is this a new component or a breaking change? Make sure you also spin up a sandbox (https://github.com/facebookincubator/create-react-app#getting-started), test it there, and tell us about it. -->
```

Comment:
Yea local import totally worked and was pretty easy. If you can think of a way to make this more prominent, I think we should. If not, no worries.

Acutally, the checklist isn't applicable to all PRs anyway... there's that whole "put an 'x' in anyway and add 'n/a' to the end of the line" section. Would you reconsider putting in there?